### PR TITLE
Migrate view bill runs page from legacy UI

### PR DESCRIPTION
**Pull request by Cruikshanks**
https://eaflood.atlassian.net/browse/WATER-4445

> Part of a series of changes to migrate the legacy view bill runs page into this project

This is the final step in migrating the legacy view bill runs page to this project. We dealt with the support services we need in previous changes. Now we need to add

- the route
- the controller handler
- the orchestration service
- the view template

We also update the menu to point to it and not the legacy `/billing/batch/list`. Worth noting (because this is the first) we use the convention of naming the handler and subsequent files `index` to reflect they deal with displaying a list of bill runs, not a single one.

Screenshot 2024-04-22 at 22 50 51

### CODE REVIEW

File: test/presenters/bill-runs/index.bill-runs.presenter.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          region: 'Albion',
+          scheme: 'sroc',
+          status: 'sent',
+          total: '£300.00',
+          type: 'Supplementary'
+        }
+      ])
+    })
+
+    describe("the 'link' property", () => {
```
Comment (Jozzey) [inline test/presenters/bill-runs/index.bill-runs.presenter.test.js:52]: Could we add a test for the link property to test the link returned when the bill run has a status of review.

**Review by Jozzey — CHANGES_REQUESTED** [review]
Just 1 comment

File: test/presenters/bill-runs/index.bill-runs.presenter.test.js (hunk 1 above)
Comment (Cruikshanks) [reply test/presenters/bill-runs/index.bill-runs.presenter.test.js:52]: 🎯 💯

File: app/presenters/bill-runs/index-bill-runs.presenter.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      id,
+      netTotal,
+      numberOfBills,
+      region,
+      scheme,
+      summer,
+      status
+    } = billRun
+
+    const type = formatBillRunType(batchType, scheme, summer)
```
Comment (Beckyrose200) [inline app/presenters/bill-runs/index-bill-runs.presenter.js:37]: Just curious what made you choose to define type here and not inside the returned object like what you have done with createdAt, link, region and total?

Comment (Cruikshanks) [reply app/presenters/bill-runs/index-bill-runs.presenter.js]: I think it comes from a refactor. I previously needed `type` for something else but now no longer do and have overlooked updating the code. Good spot!

**Review by Jozzey — APPROVED** [review]

**Pull request merged**